mv/core/src/test/scala/org/apache/carbondata/mv/rewrite/TestAllOperationsOnMV.scala

```diff
@@ -626,6 +626,32 @@ class TestAllOperationsOnMV extends QueryTest with BeforeAndAfterEach {
     sql("drop table IF EXISTS maintable")
   }
 
+  test("test duplicate column name in mv") {
```

Comment:
please can you add a couple of test cases for 
1. one column with alias , one direct column name
2. both column with alias

Reply:
added